site/gatsby-site/src/hooks/useMongo.js

```diff
@@ -0,0 +1,50 @@
+import { realmApp } from '../mongodb/authenticate';
```

Comment:
This looks like it is not the correct path. Isn't this at `site/gatsby-site/src/services/realmApp.js`?

Reply:
oh, how silly auto import, yes, correct.